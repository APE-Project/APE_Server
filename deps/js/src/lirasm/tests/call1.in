ptr = allocp 8
a = immi 65
sti2c a ptr 0
b = immi 66
sti2c b ptr 1
c = immi 67
sti2c c ptr 2
zero = immi 0
sti2c zero ptr 3
ss = calli puts cdecl ptr
nn = gei ss zero
reti nn
