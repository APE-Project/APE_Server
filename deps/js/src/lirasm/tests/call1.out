ABC
Output is: 1
